### PR TITLE
Fixes #1672 : Replace string concatenation with multi variable strings

### LINKED ISSUE #1672: TextViewBindingAdapters uses string concatenation instead of multi-variable strings
**Describe the bug**
`TextViewBindingAdapters.java` uses string concatenation instead of using proper multi-variable strings. This will cause issues in the future when we attempt to RTL translate it.

**To Reproduce**
1. Go to `TextViewBindingAdapters.java`
2. Find this line: 
```
textView.getContext().getString(R.string.profile_last_used) + " " + getTimeAgo(
 ```

### PR DESCRIPTION
**Pull request by prayutsu**
## Explanation

Fixes #1672 
Replace string concatenation with multi-variable strings format in ```TextViewBindingAdapters.java```

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by prayutsu** [comment]
@rt4914 PTAL

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -345,6 +345,7 @@
   <string name="unlink_items">Unlink Items</string>
   <string name="move_item_down">Move Item Down</string>
   <string name="move_item_up">Move Item Up</string>
+  <string name="profile_last_visited">%1$s %2$s</string>
```
Comment (rt4914) [inline app/src/main/res/values/strings.xml]: You can write this directly as `<string name="profile_last_visited">%s %s</string>`

**Review by rt4914 — CHANGES_REQUESTED** [review]
Suggest nit change.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (prayutsu) [reply app/src/main/res/values/strings.xml]: done

**Comment by prayutsu** [comment]
@rt4914 PTAL

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**